torchdynamo/convert_frame.py

```diff
@@ -396,23 +397,25 @@ def transform(instructions, code_options):
                 return None
         output_codes.add(out_code)
 
-        log.info(
+        log.log(
+            torchdynamo_logging.VERBOSE,
             format_bytecode(
                 "ORIGINAL BYTECODE",
                 code.co_name,
                 code.co_filename,
                 code.co_firstlineno,
                 code,
-            )
+            ),
         )
-        log.info(
+        log.log(
+            torchdynamo_logging.VERBOSE,
```

Comment:
Sorry, I already know you changed this once, can you call this CODE? I don't want to use verbose because we already have a config setting verbose which makes exception traces larger.

Reply:
Also consider adding all of the different levels we support under torchdynamo_logging, this will make the interface consistent for all of the log levels